packages/core/src/database/migrations/z1660599161-create-merchantRegistry.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+module.exports = {
+    up: (queryInterface, Sequelize) => {
+        return queryInterface.createTable('merchant_registry', {
+            id: {
+                type: Sequelize.INTEGER,
+                autoIncrement: true,
+                primaryKey: true,
+            },
+            name: {
+                type: Sequelize.STRING(64),
+                allowNull: false,
+            },
+            street: {
+                type: Sequelize.STRING(128),
+                allowNull: false,
+            },
+            city: {
+                type: Sequelize.STRING(64),
+                allowNull: false,
+            },
+            country: {
+                type: Sequelize.STRING(64),
+                allowNull: false,
+            },
+            gps: {
+                type: Sequelize.JSON,
+                allowNull: false,
+            },
```

Comment:
let's do like we did for the communities, just location (can be just city, or city/state), country, gps

Reply:
done!